sycl/CMakeLists.txt

```diff
@@ -184,6 +184,21 @@ install(DIRECTORY ${OPENCL_INCLUDE}/CL
   COMPONENT opencl-headers
 )
 
+if (NOT TARGET LLVMGenXIntrinsics)
```

Comment:
Please add possibility to provide pre-extracted sources.
Check existence of some cmake variable and if exist use its value as source directory. Clone repository otherwise.

Reply:
Done. Now there are 3 available ways:
- clone vc-intrinsics to llvm/projects
- set LLVMGenXIntrinsics_SOURCE_DIR
- just do nothing, vc-intrinsics will be downloaded by FetchContent